sprout/scripts/config_validation/dns_schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "All DNS records",
+  "type": "object",
+  "properties": {
+    "hostnames": {
+      "type": "array",
+      "items": {
+        "title": "Each DNS record",
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "hostname of DNS entry"
+          },
+          "records": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 1,
```

Comment:
We only allow a one element array. You might be right, but this surprised me. Can you double check?